src/MAUI/Maui.Samples/Samples/Geometry/Buffer/readme.md

```diff
@@ -32,8 +32,10 @@ Creating buffers is a core concept in GIS proximity analysis that allows you to
 
 Buffers can be generated as either `planar` (flat - coordinate space of the map's spatial reference) or `geodesic` (technique that considers the curved shape of the Earth's surface, which is generally a more accurate representation). In general, distortion in the map increases as you move away from the standard parallels of the spatial reference's projection. This map is in Web Mercator so areas near the equator are the most accurate. As you move the buffer location north or south from that line, you'll see a greater difference in the polygon size and shape. Planar operations are generally faster, but performance improvement may only be noticeable for large operations (buffering a great number or complex geometry).
 
+Geodesic buffers in the far northern and southern regions of the map will extend beyond the map's limits. The visible extent of the basemap in this sample is limited to between approximately +/-85 degrees latitude while geodesic buffers are calculated to extend all the way to the poles (+/-90 degrees). Also, since map view wraparound is active, geodesic buffers that cross the international date line (180 degrees longitude) will be [normalized](https://developers.arcgis.com/flutter/api-reference/reference/arcgis_maps/GeometryEngine/normalizeCentralMeridian.html), resulting in a multipart geometry. This results in a vertical line in the buffer graphic at the dateline.
```

Comment:
Best to point towards the .NET API reference here.

```suggestion
Geodesic buffers in the far northern and southern regions of the map will extend beyond the map's limits. The visible extent of the basemap in this sample is limited to between approximately +/-85 degrees latitude while geodesic buffers are calculated to extend all the way to the poles (+/-90 degrees). Also, since map view wraparound is active, geodesic buffers that cross the international date line (180 degrees longitude) will be [normalized](https://developers.arcgis.com/net/api-reference/api/net/Esri.ArcGISRuntime/Esri.ArcGISRuntime.Geometry.GeometryEngine.NormalizeCentralMeridian.html), resulting in a multipart geometry. This results in a vertical line in the buffer graphic at the dateline.
```

Reply:
Will need to be updated in each of the readme files. You can run `samplesync.py` to do this for you.